.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,234 @@
+name: prebuild
+
+on:
+  push:
+    branches:
+      - main
+      - dev-1.x
+      - master
+    paths:
+      - "mmdeploy/version.py"
+
+permissions: write-all
+
+jobs:
+  linux_build:
+    runs-on: [self-hosted, linux-3090]
+    container:
+      image: openmmlab/mmdeploy:manylinux2014_x86_64-cuda11.3
```

Comment:
In this image, gcc version is 10.2, which is too high.
It results in `cxxabi` issue when the prebuilt SDK is compiled under a lower gcc, e.g., 7.5